crates/eframe/src/web/web_painter_wgpu.rs

```diff
@@ -103,6 +132,7 @@ impl WebPainter for WebPainterWgpu {
         textures_delta: &egui::TexturesDelta,
     ) -> Result<(), JsValue> {
         let size_in_pixels = [self.canvas.width(), self.canvas.height()];
+        self.generate_depth_texture_view(size_in_pixels[0], size_in_pixels[1]);
```

Comment:
doing this every frame is extremely wasteful. Creating a texture has major costs!
This should only be done on startup & resize. You actually can piggy-back on the 
```rust
            if size_in_pixels[0] != self.surface_configuration.width
                || size_in_pixels[1] != self.surface_configuration.height
                || size_in_pixels[1] != self.surface_configuration.height
            {
```
check further down which resizes the backbuffer